docs/contracts/v4/guides/11-position-manager.mdx

```diff
@@ -0,0 +1,838 @@
+    bool useFeesAsLiquidity
+) external {
+```
+
+Choose the appropriate delta resolution based on whether we’re using fees:
+
+```solidity
+// Define the sequence of operations:
+    // If using fees: Handle potential fee conversion
+    // If not: Standard liquidity addition
+    bytes memory actions;
+    if (useFeesAsLiquidity) {
+        actions = abi.encodePacked(
+            Actions.INCREASE_LIQUIDITY,  // Add liquidity
+            Actions.CLOSE_CURRENCY,      // Handle token0 (might need to pay or receive)
+            Actions.CLOSE_CURRENCY       // Handle token1 (might need to pay or receive)
+        );
+    } else {
+        actions = abi.encodePacked(
+            Actions.INCREASE_LIQUIDITY,  // Add liquidity
+            Actions.SETTLE_PAIR          // Provide tokens
+        );
+    }
+```
+
+Prepare parameters based on our chosen strategy:
+
+```solidity
+// Number of parameters depends on our strategy
+    bytes[] memory params = new bytes[](
+        useFeesAsLiquidity ? 3 : 2
+    );
+
+    // Parameters for INCREASE_LIQUIDITY
+    params[0] = abi.encode(
+        tokenId,           // Position to increase
+        liquidityIncrease, // Amount to add
+        amount0Max,        // Maximum token0 to spend
+        amount1Max,        // Maximum token1 to spend
+        ""                // No hook data needed
+    );
+```
+
+Set up delta resolution parameters:
+
+```solidity
+if (useFeesAsLiquidity) {
+        // Using CLOSE_CURRENCY for automatic handling of each token
+        params[1] = abi.encode(currency0);  // Handle token0
+        params[2] = abi.encode(currency1);  // Handle token1
+    } else {
+        // Standard SETTLE_PAIR for providing tokens
+        params[1] = abi.encode(currency0, currency1);
+    }
+```
+
+Execute the operation:
+
+```solidity
+// Execute the increase
+    positionManager.modifyLiquidities(
+        abi.encode(actions, params),
+        block.timestamp + 60  // 60 second deadline
+    );
+}
+```
+
+## Decreasing Liquidity
+
+When you want to remove liquidity from a position, you’ll need to handle both the liquidity reduction and any accumulated fees. Let’s understand how to implement this effectively.
+
+### **Understanding the Operation**
+
+Decreasing liquidity involves:
+
+1. Specifying how much liquidity to remove
+2. Setting minimum amounts to receive (slippage protection)
+3. Collecting both the removed liquidity and any accumulated fees
+
+### **Delta Resolution Options**
+
+When decreasing liquidity, you’ll receive tokens, so we have different options for handling the positive deltas:
+
+1. **Standard Case**: Receive all tokens
+
+```solidity
+bytes memory actions = abi.encodePacked(
+    Actions.DECREASE_LIQUIDITY,
+    Actions.TAKE_PAIR
+);
+```
+
+**2. Dust Handling**: Ignore small amounts
+
+```solidity
+bytes memory actions = abi.encodePacked(
+    Actions.DECREASE_LIQUIDITY,
+    Actions.CLEAR_OR_TAKE,  // For token0
+    Actions.CLEAR_OR_TAKE   // For token1
+);
+```
```

Comment:
when decreasing liquidity, very rarely would you actually want to forfeit the dust...

i think we remove this case?

Reply:
You're right, removed.